dbms/src/Dictionaries/IDictionary.h

```diff
@@ -2,6 +2,8 @@
 
 #include <Core/Field.h>
 #include <Core/StringRef.h>
+#include <DataStreams/IBlockInputStream.h>
```

Comment:
You could forward declare IBlockInputStream and add `using BlockInputStreamPtr = ...`.